structure/0054-quilt-kotlin-team.md

```diff
@@ -0,0 +1,52 @@
+## Summary
+
+The Quilt Kotlin Team will be responsible for developing and maintaining the
+Quilt Kotlin Libraries (Henceforth QKL). QKL is set of libraries, built off of
+QSL, intended for use in the Kotlin programming language.

```

Comment:
Do we want to mention possible other projects? Such as PRing build tools support or a kotlin chasm frontend?

Reply:
I can do that